app/templates/eventView.html

```diff
@@ -49,7 +49,14 @@ <h3><br>Description</h3>
                         <p>This event does not require RSVP.</p>
                     {% endif %}
                     {% if eventData.recurringId %}
-                        <p>This event is a recurring event.</p>
+                        <p>This event is a recurring event.
+                            {% if eventData.nextRecurringEvent %}
+                            The next event in the series occurs on {{eventData.nextRecurringEvent.startDate.strftime("%m/%d/%Y")}}.
```

Comment:
Since this is in a sentence, let's make the date a date in english without the year. Like "occurs on December 5."